crysthack_day1_GitHub_workflow_exercise/src/main.py

```diff
@@ -1,6 +1,26 @@
+import glob
+import random
+
+
+def read_names():
+    """
+    Read the names in ../names
+    :return: names_list
+    """
+    names_list = []
+    for name_file in glob.glob("./names/*"):
+        with open(name_file, 'r') as f:
+            first_line = f.readline()
+            first_name = first_line.split()[0]
+            names_list.append(first_name)
+    return names_list
+
+
+def greet(names):
+    greeting_list = ["Hello", "Hi", "Howdy","Hola","Ciao"]
```

Comment:
generally in my group we avoid putting the type into the variable name unless it is important for carrying information.  This is for readability (and to avoid accidentally defining a variable with the name of a type...).  Our goal is always to make the code self documenting.  Developer time is much more valuable than cpu-time in most cases (except where performance is really a bottle-neck). If I read
```
greetings = ["Hello", "Hi", "Howdy", "Hola", "Ciao"]
```
then later (you don't use a for loop but this is an example)
```
for greeting in greetings:
```
I know exactly your intent even though you wrote the code thousands of miles away....it is like you talking to me.